libs/web/api/src/lib/test-environment.interface.ts

```diff
@@ -1,6 +1,8 @@
 export interface TestEnvironment {
   readonly id?: string;
   readonly artifactId: string;
+  artifactName: string;
```

Comment:
readonly?

Reply:
Yes. The reason for that is, that the artifactName and certificateName will be later initialized inside the `test-environment.ts`. This is only to display information on the frontend. 
For this reason we implemented an additional TestEnvironmentDto inside `test-environment.interface.ts` to clarify the process now.